pandas/tests/io/test_s3.py

```diff
@@ -6,3 +10,16 @@ class TestS3URL(object):
     def test_is_s3_url(self):
         assert is_s3_url("s3://pandas/somethingelse.com")
         assert not is_s3_url("s4://pandas/somethingelse.com")
+
+
+def test_streaming_s3_objects():
+    pytest.importorskip('botocore', minversion='1.10.47')
```

Comment:
can you add the issue number here as a comment

Reply:
yup, done